mlrun/projects/project.py

```diff
@@ -3899,7 +3899,9 @@ def store_alert_config(self, alert_data: AlertConfig, alert_name=None):
         db = mlrun.db.get_run_db(secrets=self._secrets)
         if alert_name is None:
             alert_name = alert_data.name
-        return db.store_alert_config(alert_name, alert_data.dict(), self.metadata.name)
+        return db.store_alert_config(
+            alert_name, alert_data.to_dict(), self.metadata.name
```

Comment:
```suggestion
            alert_name, alert_data, project=self.metadata.name
```
Don't really need the `to_dict()` here now, right?
Also, since you're passing 2 names here it would be helpful to explicitly state which parameter is what (at least for the project case).